recipes/pyhdk/meta.yaml

```diff
@@ -0,0 +1,150 @@
+{% set version = "0.1.0" %}  # PEP 386
+{% set source_version = "0.1" %}
+{% set base_version = "0.1.0" %}
+{% set number = "0" %}
+{% set cuda_enabled = cuda_compiler_version is not undefined and cuda_compiler_version == '11.0' %}
+{% set build_ext_version = "1.0.0" %}
+{% set build_ext = "cuda" if cuda_enabled else "cpu" %}
+{% set build_string = "h{}_{}".format(PKG_HASH, number) %}
+{% set build_string_ext = "{}_{}".format(build_string, build_ext) %}
+{% set py_build_string_ext = "py{}{}_{}".format(CONDA_PY, build_string, build_ext) %}
+{% set py_build_string = "py{}{}".format(CONDA_PY, build_string) %}
+{% set install_base = "opt/hdk" %}
+{% set arrow_version = "6.*" %}
+{% set arrow_proc_version = "3.*" %}
+{% set llvm_version = "12" %}
+
+package:
+  name: hdk
+  version: {{ version }}
+
+source:
+  url: https://github.com/intel-ai/hdk/releases/download/v{{ source_version }}/v{{ source_version }}.full.tar.gz
+  sha256: 3de25732870b384a0cfd28009d29999d44f0d8ea4f32add8d3ba810fa56eb0de
+
+build:
+  number: {{ number }}
+  skip: true  # [cuda_compiler_version not in (undefined, "None", "11.0")]
+
+outputs:
+  
+  - name: pyhdk
+    version: {{ version }}
+    script: build-hdk.sh
+    build:
+      string: {{ py_build_string_ext }}
+      skip: true  # [cuda_compiler_version not in (undefined, "None", "11.0") or not linux64]
+      missing_dso_whitelist:
+        - '*/libcuda.*'  # [cuda_compiler_version not in (undefined, "None")]
+      track_features:
+        {{ "- arrow-cuda" if cuda_enabled else "" }}
+      missing_dso_whitelist:
+        - '*/libjvm.so'
+      ignore_run_exports:
+        - arrow-cpp {{ arrow_version }}        
```

Comment:
Could you please add a comment about why these are necessary? many of these are not in your host section, and thus wouldn't be introducing run_exports anyway.

Reply:
Hi Mike, I've deleted the section, see the full answer below